php/auth.php

```diff
@@ -0,0 +1,49 @@
+<?php
+$ERRORLOG = getenv('PHP_ERROR_LOG');
```

Comment:
isn't lighttpd still the default web server for pi-hole core or did they switch to NGINX and I missed that?  I was thinking /var/log/lighttpd/error.log (mixed in with the lighttpd log, I formatted the date string the same as lighttpd I believe)